modules/local/sentieon/bwamem.nf

```diff
@@ -9,7 +9,7 @@ process SENTIEON_BWAMEM {
     tuple val(meta), path(reads)
     path fasta
     path fai
-    path index
+    tuple val(meta2), path(index)
```

Comment:
something more detailed than meta2?

Reply:
I call it meta2 because that's what it is called in bwamem2 module.

Reply:
Ok, then maybe a comment about what is contains/how it relates to meta to help development?

Reply:
Good point! Will add a comment 👍🏻